server/liveblog/posts/posts.py

```diff
@@ -218,6 +218,9 @@ def on_update(self, updates, original):
             # if you publish a post it will save a published date and register who did it
             updates['published_date'] = utcnow()
             updates['publisher'] = getattr(flask.g, 'user', None)
+            # if you publish a post and hasn't `content_updated_date` add it.
+            if not updates.get('content_updated_date', False):
+                updates['content_updated_date'] = updates['published_date']
```

Comment:
the same question here: why do we need the updated_date to be the same as publihed_date?